lib/tasks/refresh_materialized_db_views.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+desc 'Refresh materialized views for dashboards'
+task refresh_materialized_db_views: :environment do
+  # LatestBloodPressuresPerPatientPerMonth should be refreshed before
+  # LatestBloodPressuresPerPatientPerQuarter and LatestBloodPressuresPerPatient
+
+  Rails.logger.info 'Refreshing LatestBloodPressuresPerPatientPerMonth'
+  LatestBloodPressuresPerPatientPerMonth.refresh
+
+  Rails.logger.info 'Refreshing LatestBloodPressuresPerPatient'
+  LatestBloodPressuresPerPatient.refresh
+
+  Rails.logger.info 'Refreshing LatestBloodPressuresPerPatientPerQuarter'
+  LatestBloodPressuresPerPatientPerQuarter.refresh
```

Comment:
From a dev's UX perspective, it's nice to have a `Rails.logger.info "Refresh complete"` at the very end. That way, when the task takes a long time, I don't have to wonder whether the last step takes a while, or whether the script hung or failed to terminate correctly after execution.